libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -299,6 +299,12 @@ bool NameAndTypeResolver::resolveNamesAndTypesInternal(ASTNode& _node, bool _res
 			if (!resolveNamesAndTypesInternal(*baseContract, true))
 				success = false;
 
+		if (
+			StorageLayoutSpecifier* storageLayoutSpecifier = contract->storageLayoutSpecifier()
+		)
```

Comment:
```suggestion
		if (StorageLayoutSpecifier* storageLayoutSpecifier = contract->storageLayoutSpecifier())
```

nit: I don't think the condition is long enough to warrant a break.